eng/pipelines/merge-docs-main-to-live.yml

```
@@ -0,0 +1,86 @@
+parameters:
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+- name: DocsRepos
+  type: object
+  default:
+    - RepoName: azure-docs-sdk-java
+      RepoOwner: Azure
+    - RepoName: azure-docs-sdk-python
+      RepoOwner: MicrosoftDocs
+    - RepoName: azure-docs-sdk-node
+      RepoOwner: MicrosoftDocs
+
+- name: BaseBranch
+  type: string
+  default: live
+
+jobs:
+  - job: SyncUp
+    pool:
+      vmImage: windows-2019
+    variables:
+      DocRepoLocation: $(Pipeline.Workspace)/docs
+    steps:
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+        parameters:
+          Paths: 
+          - /eng/pipelines/branch-syncup.yml
+      - ${{ each repo in parameters.DocsRepos }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+            SkipDefaultCheckout: true
+            Repositories:
+            - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+              WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
```

`/eng` is included by default, so the first checkout can be simplified. Additionally you can consolidate the checkout steps into one to make it easier to scroll through the step display output in the pipeline UI. I think something like this should work if you are interested.

```suggestion
      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
        parameters:
          SkipDefaultCheckout: true
          Repositories:
          - Name: $(Build.Repository.Name)
            Commitish: $(Build.SourceVersion)
            WorkingDirectory: $(System.DefaultWorkingDirectory)
          - ${{ each repo in parameters.DocsRepos }}:
            - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
              WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
```